src/main/scala/centaur/test/workflow/WorkflowData.scala

```diff
@@ -3,13 +3,38 @@ package centaur.test.workflow
 import java.nio.file.Path
 
 import cats.data.Validated._
+import centaur.CentaurConfig
 import centaur.test._
 import com.typesafe.config.Config
 import configs.Result
 import configs.syntax._
+import spray.json._
+
 
 case class WorkflowData(wdl: String, inputs: Option[String], options: Option[String])
 
+case class WorkflowOptions(options: Option[String]) {
+  import DefaultJsonProtocol._
+
+  def insertSecrets(): Option[String] = {
```

Comment:
When I look at this I feel like it should return a `WorkflowOptions` with the modified `options` field. 

That said if you changed the function name to be something like `optionsWithSecretsInserted` it'd dial down my pedantic-ness and I'd not really care